RFC/src/RFC-0311_AssetTemplates.md

```diff
@@ -107,8 +107,8 @@ The instruction is in JSON format and MUST contain the following fields:
 | **template-specific data**         | Object        | Template-specific metadata can be defined in this section                                                                   |
 | **Signatures**                     |               |                                                                                                                             |
 | metadata_hash                      | `u256`        | A hash of the previous three sections' data, in canonical format (`m`)                                                      |
-| creator_sig                        | `u256`        | A digital signature of the message `H(R || P || m)` using the asset creator’s private key corresponding to the `issuer` PKH |
-| raid_signature                     | `u256`        | Digital signature for the given RAID, signing the message `H(R || P || RAID_ID || m)`                                       |
+| creator_sig                        | `u256`        | A digital signature of the message `H(R ‖ P ‖ m)` using the asset creator’s private key corresponding to the `issuer` PKH |
+| raid_signature                     | `u256`        | Digital signature for the given RAID, signing the message `H(R ‖ P ‖ RAID_ID ‖ m)`                                       |
```

Comment:
Some confusion about this raid_signature:
- This is not the same as the RAID_ID signature described in RFC_0301 due to the inclusion of `m`.
- Therefore the `creator_sig` can just become the `raid_signature` and there is no need for both.

Reply:
You have to sign something that links the signature to _this asset_, otherwise you could copy and paste signatures from other places and spoof them

Reply:
Agreed.
My confusion: Is the intention that this `raid_signature` be the same as the RAID_ID signature in the OpenAlias TXT DNS record? If not then `H(R ‖ P ‖ RAID_ID ‖ m)` actually can replace `H(R ‖ P ‖ m)` used for the `creator_sig`